### PR TITLE
Improved the susi_ai android section

### LINKED ISSUE #318: Add and correct information for section "SUSI.AI Android App" and add app image
The new section "SUSI.AI Android App" needs correct info, links and app image. Please help to do it.

### PR DESCRIPTION
**Pull request by Sid911**
**Changes Made In the Pull**
- Added images for susi ai (android and IOS)
- custom made image for android ... made by myself 
- edited the susi ai android section
- added links

Preview link : https://sid911.github.io/labs/
### Fixes #318

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -407,12 +407,12 @@ <h1>SUSI.AI Android App</h1>
...
-                              <a href="https://www.facebook.com/pslabapp/" target="_self" title="Facebook Profile" class="fa fa-facebook"></a>
-                              <a href="https://twitter.com/pslabapp" target="_self" title="Twitter Profile" class="fa fa-twitter"></a>
+                              <a href="https://chat.susi.ai/overview" target="_self"  title="Website" class="fa fa-link"></a>
+                              <a href="https://groups.google.com/forum/#!forum/susiai" target="_self"  title="Mailing List" class="fa fa-envelope"></a>
+                              <a href="https://gitter.im/fossasia/susi_android?source=orgpage" target="_self" title="Chat Channel" class="fa fa-comments"></a>
+                              <a href="https://github.com/fossasia/susi_android" target="_self" title="Contribute" class="fa fa-github"></a>
+                              <a href="https://www.facebook.com/asksusiai/" target="_self" title="Facebook Profile" class="fa fa-facebook"></a>
+                              <a href="https://twitter.com/susiai_?lang=en" target="_self" title="Twitter Profile" class="fa fa-twitter"></a>
```
Comment (jogendra) [inline index.html]: replace it with https://twitter.com/susiai_

File: index.html [hunk 2]
```diff
@@ -407,12 +407,12 @@ <h1>SUSI.AI Android App</h1>
...
-                              <a href="https://groups.google.com/group/fossasia-pslab" target="_self"  title="Mailing List" class="fa fa-envelope"></a>
-                              <a href="https://gitter.im/fossasia/pslab" target="_self" title="Chat Channel" class="fa fa-comments"></a>
-                              <a href="https://github.com/fossasia?utf8=✓&q=pslab" target="_self" title="Contribute" class="fa fa-github"></a>
-                              <a href="https://www.facebook.com/pslabapp/" target="_self" title="Facebook Profile" class="fa fa-facebook"></a>
-                              <a href="https://twitter.com/pslabapp" target="_self" title="Twitter Profile" class="fa fa-twitter"></a>
+                              <a href="https://chat.susi.ai/overview" target="_self"  title="Website" class="fa fa-link"></a>
+                              <a href="https://groups.google.com/forum/#!forum/susiai" target="_self"  title="Mailing List" class="fa fa-envelope"></a>
+                              <a href="https://gitter.im/fossasia/susi_android?source=orgpage" target="_self" title="Chat Channel" class="fa fa-comments"></a>
```
Comment (jogendra) [inline index.html]: replace it with https://gitter.im/fossasia/susi_android

**Comment by Sid911** [comment]
@jogendra  Done sir...

**Comment by jogendra** [comment]
@Sid911 Please provide a preview link

**Comment by Sid911** [comment]
@jogendra Added the preview link 😊

**Comment by Sid911** [comment]
@mariobehling Is this good enough to be merged?

**Pull request merged**